src/routes/about/Card.svelte

```diff
@@ -1,10 +1,13 @@
 <script>
-    import src from '$lib/lino-sablay.svg';
+    export let src = '$lib/lino-sablay.svg';
+    export let bgStyles = 'bg-csi-neutral-200 rounded-t-full rounded-b-xl';
```

Comment:
I believe this isn't necessary at the moment. Let's reintroduce it only when needed. 👌 
```suggestion
```